packages/flame/src/Text/Text.tsx

```diff
@@ -174,14 +174,14 @@ Heading3.defaultProps = {
 export const Heading4 = styled(BaseText)<TextProps>`
   font-size: ${themeGet('fontSizes.xsmall')};
   line-height: ${themeGet('lineHeights.3')};
+  text-transform: uppercase;
 `.withComponent('h4');
 
 Heading4.defaultProps = {
   ...BaseText.defaultProps,
   fontWeight: 'bold',
   color: 'textHeading',
   letterSpacing: 3,
-  textTransform: 'uppercase',
```

Comment:
This cannot be overwritten anyways, so move it to the styled definition.
This also fixes that super weird issue with docgen.